website/docs/docs/build/metricflow-core-concepts.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "About to MetricFlow"
+id: metricflow-core-concepts
+description: "Learn more about MetricFlow and its key concepts"
+sidebar_label: About to MetricFlow
+tags: [Metrics, Semantic Layer]
+---
+
+
+This guide introduces MetricFlow's fundamental ideas for new users. MetricFlow helps you define and manage logic behind your company's metrics. It's an opinionated set of abstractions and helps data consumers retrieve metric datasets from a data platform quickly and efficiently.
+
+There are a few key principles:
+
+- **Flexible, but complete** &mdash;  Ability to create any metric on any data model by defining logic in flexible abstractions.
+- **Don't Repeat Yourself (DRY)** &mdash; Avoid repetition by allowing metric definitions to be enabled whenever possible.
+- **Simple with progressive complexity** &mdash; Make MetricFlow approachable by relying on known concepts and structures in data modeling. Create escape hatches that enable folks to pursue more advanced and unsupported features.
+- **Performant and efficient** &mdash; Allow for performance optimizations in centralized data engineering while still enabling distributed definition and ownership of logic.
+
+## Framework
+
+- MetricFlow is a semantic layer that helps you create metrics by constructing appropriate queries for different granularities and dimensions that are useful for various business applications. 
+
+- It uses YAML files to define a semantic graph that acts like a data map. This graph consists of [semantic models](/docs/build/semantic-models), which serve as data entry points, and [metrics](/docs/build/metrics-overview), which are functions used to create new quantitative indicators.
```

Comment:
`data map` do we use this term anywhere else? Not sure I understand what it is attempting to convey. Maybe we switch to "maps language to data"?